resources/factions/uk_1990.json

```diff
@@ -55,7 +55,7 @@
     "FFG Oliver Hazard Perry",
     "DDG Arleigh Burke IIa",
     "CG Ticonderoga",
-    "LHA-1 Tarawa",
+    "R05 Invincible"
```

Comment:
Comma missing from here?

Reply:
Thanks, yeah! I removed the Stennis when I tested it but then re-added it but forgot the comma... doh!

Reply:
Please add the comma, otherwise we can't merge this since it wouldn't be valid json.

Reply:
D'oh, thanks, fixed. Sorry for being such a nub.